src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+.. _Authorization:
+
+Authorization
+=============
+
+Cylc supports multi user access. You can grant control of your
```

Comment:
Would be good to clarify the layer of the system this comes from to set expectations (e.g. ATM the CLI doesn't support this):

```suggestion
The Cylc UI Server supports multi user access. You can grant control of your
```